modules/core/shared/src/main/scala/lucuma/core/validation/InputValidWedge.scala

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.validation
+
+import cats.data.NonEmptyChain
+import cats.syntax.all._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.numeric.Positive
+import eu.timepit.refined.types.numeric.PosBigDecimal
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.optics._
+import monocle.Iso
+import monocle.Prism
+import singleton.ops._
```

Comment:
Singleton ops is no longer available on scala 3. Could we consider an alternative?

Reply:
Yup, I'm going to try to implement what Todd suggested. Maybe let's hold the review until then.

Reply:
See the comment above. I don't think it is necessary anymore.